web/fido_app/templates/profile.html

```diff
@@ -17,11 +17,31 @@
 {% endblock %}
 
 {% block cardBody %}
+<p>
+  <strong>Total Credit Earned by Logging in: </strong>
+  <span>${{ "%0.2f" % get_credit(current_user.login_bitfield) }}</span>
+</p>
+
+<p><strong>Remaining Daily Tasks</strong></p>
+{% if current_user.last_complete_login != date.today() %}
+<ul>
+  {% if session['logged_in_today'] != 'password' %}
+    <li>Login with Passwords</li>
+  {% endif %}
+  {% if session['logged_in_today'] != 'fido2' %}
+    <li>Login with WebAuthn</li>
+  {% endif %}
+</ul>
+{% else %}
+<span>✅ Completed all daily tasks!</span>
+{% endif %}
+
+<!-- 
```

Comment:
Seems good to me. The only thing I'd add is an explicit message saying the user is done for the day if they've logged in both times already; e.g., "No tasks remaining today. Come back tomorrow!" instead of "Remaining Daily Tasks".